security/deployment.adoc

```diff
@@ -128,34 +128,39 @@ There are two types of signature transports.
 - `atomic`: Managed by the {product-title} API.
 - `docker`: Served as a local file or by a web server.
 
-Signatures using the `atomic` transport type are managed by the {product-title}
-API. Images must be stored by the OpenShift Container Registry. No additional
-configuration is required due to the *docker/distribution*
-xref:../admin_guide/image_signatures.adoc#reading-image-signatures-via-registry-api[`extensions` API] to auto-discover the image signature endpoint.
-
-Signatures using the `docker` transport type are served by local file or web
-server. They have the benefit of the most flexibility: any container registry
-can be used to serve images with an independent server to deliver binary
-signatures.
-
-However, the `docker` transport type requires additional configuration. Each
-node must be configured with the URI of the signature server using
-arbitrarily-named YAML files placed into a directory on the host system,
+The {product-title} API manages signatures that use the `atomic` transport type.
+You must store the images that use this signature type in the 
+the OpenShift Container Registry. Because the *docker/distribution* 
+xref:../admin_guide/image_signatures.adoc#reading-image-signatures-via-registry-api[`extensions` API]
+auto-discovers the image signature endpoint, no additional
+configuration is required.
+
+Signatures that use the `docker` transport type are served by local file or web
+server. These signatures are more flexible: you can serve images from any 
+container registry and use an independent server to deliver binary
+signatures. According to the
+link:https://github.com/containers/image/blob/master/docs/signature-protocols.md[Signature access protocols],
+you access the signatures for each image directly; the root of the server
+directory does not display its file structure.
+
+However, the `docker` transport type requires additional configuration. You must
+configure node with the URI of the signature server by placing
+arbitrarily-named YAML files into a directory on the host system,
```

Comment:
This is a legacy issue, but do you know if the "arbitrarily-named" YAML file same as the "configuration files" that contain the URI/sigstore?

Reply:
That's the way I read it. I can change "configuration files" in the next sentence to "YAML configuration files," if you think that would help.